tests/GridFS/UnusableStream.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace MongoDB\Tests\GridFS;
+
+final class UnusableStream
```

Comment:
I don't recall if this was in the original PR (missed it if so), but great idea!